### ISSUE TITLE
Switching between levels resets current palette selection.

### ISSUE THREAD

**Issue by MisterABK** [opentoonz/opentoonz#1757]
I often find myself switching from one level to another to tweak something, and when I come back, the brush has reverted to palette 0 (black) instead of the colour I was drawing with before. Since the palettes seem to be attached to levels, would it be better if the brush remembered which colour was selected in each level?

**Comment by gab3d** [comment]
I tend to agree with this point of view as it seems more logical, workflow-wise. 
(at least within the same working session).

**Comment by artisteacher** [comment]
I agree!

**Comment by DarrenTAnims** [comment]
I strongly agree. This is frustrating when using "frame flipping" to do inbetweening and then having to re-select your brush. I've got to the point now where I try to use brush index 1 as my main brush, just to stop this issue.

**Comment by gab3d** [comment]
I agree too.
it seems logical and practical.
and it doesn't seem to be impossible to implement either.

**Comment by RodneyBaker** [comment]
> if I'm drawing with colour 10 and switch to another level (or a frame with no level attached) and come back, then it reverts to colour 0. Why not remember that I was using 10?

This is the essence of the problem and had an open report... can't find it at present.
It on that fine line of bug/enhancement but is definitely something that users want to see fixed.

> it doesn't seem to be impossible to implement 

Nothing (well... very few things at least) are impossible to implement.  :)

**Comment by RodneyBaker** [comment]
It's one of those wonderful bugs/UX (workflow) enhancements that skirts the line.
It's a process that needs updating although operating as designed but certainly not optimally.
I've tagged as bug so it can rate a higher priority when we kick off the review.
